astroid/nodes/node_classes.py

```diff
@@ -2016,13 +2020,17 @@ def postinit(self, nodes: typing.List[NodeNG]) -> None:
         """
         self.nodes = nodes
 
-    def scope(self):
+    def scope(self) -> "LocalsDictNodeNG":
         """The first parent node defining a new scope.
 
         :returns: The first parent scope node.
         :rtype: Module or FunctionDef or ClassDef or Lambda or GenExpr
```

Comment:
Is a ``LocalsDictNodeNG`` the same as ``Module or FunctionDef or ClassDef or Lambda or GenExpr``? 
```suggestion
```

Reply:
Yes! All classes which can considered `scopes` inherit from this base class!

Reply:
I have added it to the docstring